docs/source/getting_started/writing_code.ipynb

```diff
@@ -225,9 +225,9 @@
    "source": [
```

Comment:
<p>Call it <code>pre-built workspace</code> instead of Stack.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I am not clear on this. In the interface while you're creating a workspace, these are called Stacks. Is there an implicit request to change the name in the interface? Is the term pre-built workspace used elsewhere?

Reply:
the word Stacks is not in the interface anywhere I can see
![image](https://github.com/MAAP-Project/maap-documentation/assets/306179/891b32ab-b561-4781-91ac-3a15e9e5d570)

Reply:
You're right, it's not on that panel. But it's on several others... Seems like we should clean it up a bit :D

In the Getting Started guide I also defined Stack. I don't mind getting rid of it, but we should make it consistent everywhere.

<img width="1021" alt="Screenshot 2023-09-28 at 3 06 18 PM" src="https://github.com/MAAP-Project/maap-documentation/assets/1156193/91b796a6-9389-431f-86ca-9ffe9909b3b3">

<img width="953" alt="Screenshot 2023-09-28 at 3 06 56 PM" src="https://github.com/MAAP-Project/maap-documentation/assets/1156193/86f4cfd1-33f6-4877-8b55-673a0cf4d02e">

Reply:
In your screenshot it says "Select a Sample"... what does that mean? Maybe that's supposed to say Select a Stack?

Reply:
And then in the "Custom Workspace" area it's referred to as a "devfile".

Reply:
Yup, this is super confusing.

Reply:
Is it okay to leave it as "Stack" for now and make a new issue to clean up the docs and make this consistent everywhere?

Reply:
https://github.com/MAAP-Project/maap-documentation/issues/343